qadi_simple.py

```
@@ -0,0 +1,342 @@
+                    
+                    # Update total time and cost
+                    total_time = elapsed_time + evolution_time
+                    print("\n" + "═" * 50)
+                    print(f"✅ Total time (QADI + Evolution): {total_time:.1f}s")
+                    print(f"💰 Total cost: ${result.total_llm_cost:.4f}")
+                else:
+                    print(f"\n❌ Evolution failed: {evolution_result.error_message}")
+                    
+            except ImportError:
+                print("\n❌ Evolution modules not available. Please check installation.")
+            except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)